src/resources/views/create.blade.php

```diff
@@ -3,7 +3,8 @@
 @section('header')
 	<section class="content-header">
 	  <h1>
-	    {{ trans('backpack::crud.add') }} <span>{{ $crud->entity_name }}</span>
+        <span class="text-capitalize">{{ $crud->entity_name_plural }}</span>
```

Comment:
I'd prefer not to do this - what if `text-capitalize` is not what we/they want?

Reply:
Well it would look bad without it. I don't think there's a language where headings can start with an undercase letter - this is just to make it look pretty and consistent across actions.